src/custom-types.json

```diff
@@ -257,6 +257,14 @@
       "http://www.iana.org/assignments/media-types/application/vnd.sun.wadl+xml"
     ]
   },
+  "application/wasm": {
+    "compressible": true,
+    "extensions": ["wasm"],
+    "notes": "WebAssembly module",
+    "sources": [
+      "https://webassembly.github.io/spec"
```

Comment:
Yea, definitely can get it added. Can you add a link to a page that at least mentions the file extension `.wasm` and the MIME type `application/wasm`? The page here is long, but skimming through it I didn't see either mentioned.